build/azure-devops/scraper-ci.yml

```diff
@@ -77,6 +77,20 @@ stages:
          solutionOrProjectPath: 'src/Promitor.sln'
          failBuildLevelSelector: 'Warning'
          commandLineInterfacePath: '$(Build.SourcesDirectory)/Lib/Resharper'
+- stage: Linting
+  displayName: Linting
+  dependsOn: []
+  jobs:
+   - job: RunLinters
+     displayName: Run Linters
+     condition: succeeded()
+     pool:
+       vmImage: ubuntu-16.04
+     steps:
+     - script: npm install
+       displayName: 'Install npm Packages'
+     - script: npm run --silent markdownlint
```

Comment:
What does the `--silent` do?

Reply:
It's just to get rid of the noise that npm outputs when a command fails. It's easier to explain with an example, so here's what it's like without `--silent`:

```shell
 $ npm run markdownlint                                                       /c/Users/AdamConnelly/github.com/adamrpconnelly/promitor

> promitor@1.0.0 markdownlint C:\Users\AdamConnelly\github.com\adamrpconnelly\promitor
> markdownlint -c .markdownlint.json --ignore node_modules .

README.md: 28: MD013/line-length Line length [Expected: 80; Actual: 142]
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! promitor@1.0.0 markdownlint: `markdownlint -c .markdownlint.json --ignore node_modules .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the promitor@1.0.0 markdownlint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\AdamConnelly\AppData\Roaming\npm-cache\_logs\2019-10-12T16_05_15_723Z-debug.log
```

But if we include the `--silent` flag, we just get the actual errors output, and we don't get all the `npm ERR!` lines that don't really add any value:

```shell
$ npm run --silent markdownlint                                            /c/Users/AdamConnelly/github.com/adamrpconnelly/promitor 1
README.md: 28: MD013/line-length Line length [Expected: 80; Actual: 142]
```

Reply:
Sounds good, thanks!